bin/fin

```diff
@@ -3156,35 +3065,41 @@ system_status ()
 }
 
 # Starts all Docksal related things
-# TODO: add projects, network, /etc/exports, etc. here for a complete shutdown of Docksal
 system_start ()
 {
 	if ! is_linux && ! is_docker_native && ! is_wsl; then
-		echo "Starting Docksal VM..."
-		vm start
-		system_reset
-	else
-		echo "Starting Docksal system services..."
-		system_reset
+		if is_docker_machine_running; then
+			echo "Machine ${DOCKER_MACHINE_NAME} is already running."
+		else
+			echo "Starting Docksal VM..."
+			docker_machine_start
+		fi
 	fi
+
+	echo "Starting Docksal system services..."
+	system_reset
 }
 
 # Stops/disables all Docksal things
 system_stop ()
 {
 	if ! is_linux && ! is_docker_native && ! is_wsl; then
-		echo-green "Stopping Docksal VM..."
-		# VM stop will perform configure_resolver off
-		vm stop
+		# Stop the docker-machine VM.
+		if is_docker_machine_running; then
```

Comment:
KISS principle says there is no need for this check here. Because if it is stopped docker-machine will handle it itself, but at the same time is there is some bug with machine status detection it can prevent this from stopping the machine. 
So no gains, and a loss in possible bug prevention. 
I would leave this check out.